### PR TITLE
use new function instead deprecated function

### PR DESCRIPTION
**Pull request by cl2017**
Signed-off-by: cl2017 <chenlin.liu@daocloud.io>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
use new function instead deprecated function

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/edgecontroller/controller/upstream.go [hunk 1]
```diff
@@ -138,7 +138,7 @@ func (uc *UpstreamController) Start() error {
 	go uc.dispatchMessage()
 
 	for i := 0; i < int(uc.config.Load.UpdateNodeStatusWorkers); i++ {
-		go uc.updateNodeStatus()
+		go uc.patchNode()
```
Comment (fisherxu) [inline cloud/pkg/edgecontroller/controller/upstream.go:141]: We already have a `patchNode` at L168, this one is used to keep compatible with old version of edgecore. And will be removed in future release. @cl2017

File: cloud/pkg/controllermanager/controllermanager.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func NewAppsControllerManager(ctx context.Context) (manager.Manager, error) {
 }
 
 func setupControllers(ctx context.Context, mgr manager.Manager) error {
-	Serializer := json.NewYAMLSerializer(json.DefaultMetaFactory, appsScheme, appsScheme)
+	Serializer := json.NewSerializerWithOptions(json.DefaultMetaFactory, appsScheme, appsScheme, json.SerializerOptions{Yaml: true})
```
Comment (fisherxu) [inline cloud/pkg/controllermanager/controllermanager.go:51]: cc @Congrool @vincentgoat ptal.

File: cloud/pkg/edgecontroller/controller/upstream.go (hunk 1 above)
Comment (cl2017) [reply cloud/pkg/edgecontroller/controller/upstream.go:141]: ok, I'll revert this change

File: cloud/pkg/controllermanager/controllermanager.go (hunk 2 above)
Comment (Congrool) [reply cloud/pkg/controllermanager/controllermanager.go:51]: LGTM, they are equivalent I think.

Comment (gy95) [reply cloud/pkg/controllermanager/controllermanager.go:51]: LGTM, please squalish the two commits to one

Comment (cl2017) [reply cloud/pkg/controllermanager/controllermanager.go:51]: ok, done @fisherxu @Congrool @gy95

**Review by vincentgoat — COMMENTED** [review]
lgtm

**Review by fisherxu — COMMENTED** [review]
/approve

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**